### PR TITLE
feat(IntegrationDirectory): Add acceptance test for document-based integrations

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective
Add percy snapshot tests for document-based integrations.

Screen Shot 2020-04-09 at 3 04 38 PM

### CODE REVIEW

**Comment by manuzope** [comment]
Can we do this via percy? My thinking is that it might give us wider coverage of possible issues.

**Comment by NisanthanNanthakumar** [comment]
> Can we do this via percy? My thinking is that it might give us wider coverage of possible issues.

@manuzope this will send it to percy.

File: tests/acceptance/test_organization_document_integration_detailed_view.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class OrganizationDocumentIntegrationDetailView(AcceptanceTestCase):
+    """
+    As a developer, I can view an document-based integration, and learn more about it with the linked resources.
+    """
+
+    def setUp(self):
+        super(OrganizationDocumentIntegrationDetailView, self).setUp()
+        self.login_as(self.user)
+
+    def load_page(self, slug, configuration_tab=False):
```
Comment (scefali) [inline tests/acceptance/test_organization_document_integration_detailed_view.py:15]: @NisanthanNanthakumar You don't need the logic for `configuration_tab` here

Comment (scefali) [reply tests/acceptance/test_organization_document_integration_detailed_view.py:15]: @NisanthanNanthakumar You still have the parameter `configuration_tab`

Comment (NisanthanNanthakumar) [reply tests/acceptance/test_organization_document_integration_detailed_view.py]: @scefali missed that! updated.

**Review by scefali — APPROVED** [review]

**Pull request merged**